c2corg_api/jobs/purge_expired_tokens.py

```diff
@@ -0,0 +1,22 @@
+from c2corg_api.models.token import Token
+from datetime import datetime
+from sqlalchemy.orm import sessionmaker
+
+import logging
+log = logging.getLogger(__name__)
+
+
+def purge_token(test_session=None):
+    now = datetime.utcnow()
+    session = sessionmaker()() if not test_session else test_session
+
+    try:
+        count = session.query(Token).filter(
+                Token.expire <= now).delete(synchronize_session=False)
```

Comment:
Have you considered creating an index for `expire` to avoid sequential scans? Maybe also for the other query.